azurerm/data_source_arm_container_registry.go

```diff
@@ -0,0 +1,109 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmContainerRegistry() *schema.Resource {
+	return &schema.Resource{
+		Read:          dataSourceArmContainerRegistryRead,
+		MigrateState:  resourceAzureRMContainerRegistryMigrateState,
+		SchemaVersion: 2,
```

Comment:
can we remove these two lines `MigrateState` / `SchemaVersion` - since they're not needed?